docs/api.rst

```diff
@@ -2782,69 +2782,129 @@ Querying onchain balances
-                       "assets": {
-                           "ETH2": {"amount": "32.45", "usd_value": "40241.55"},
-                       },
-                  }},
+        "result":{
+          "per_account":{
+            "BTC":{
+              "standalone":{
+                "3Kb9QPcTUJKspzjQFBppfXRcWew6hyDAPb":{
+                  "amount":"0.5",
+                  "usd_value":"3770.075"
+                },
+                "33hjmoU9XjEz8aLxf44FNGB8TdrLkAVBBo":{
+                  "amount":"0.5",
+                  "usd_value":"3770.075"
+                }
               },
-              "totals": {
-                  "assets": {
-                      "BTC": {"amount": "1", "usd_value": "7540.15"},
-                      "ETH": {"amount": "10", "usd_value": "1650.53"},
-                      "ETH2": {"amount": "65.57", "usd_value": "85484.76"},
-                      "eip155:1/erc20:0x6B175474E89094C44Da98b954EedeAC495271d0F": {"amount": "15", "usd_value": "15.21"}
+              "xpubs":[
+                {
+                  "xpub":"xpub68V4ZQQ62mea7ZUKn2urQu47Bdn2Wr7SxrBxBDDwE3kjytj361YBGSKDT4WoBrE5htrSB8eAMe59NPnKrcAbiv2veN5GQUmfdjRddD1Hxrk",
+                  "derivation_path":"m/0/0",
+                  "addresses":{
+                    "1LZypJUwJJRdfdndwvDmtAjrVYaHko136r":{
+                      "amount":"0.5",
+                      "usd_value":"3770.075"
+                    },
+                    "1AMrsvqsJzDq25QnaJzX5BzEvdqQ8T6MkT":{
+                      "amount":"0.5",
+                      "usd_value":"3770.075"
+                    }
+                  }
+                },
+                {
+                  "xpub":"zpub6quTRdxqWmerHdiWVKZdLMp9FY641F1F171gfT2RS4D1FyHnutwFSMiab58Nbsdu4fXBaFwpy5xyGnKZ8d6xn2j4r4yNmQ3Yp3yDDxQUo3q",
+                  "derivation_path":"m",
+                  "addresses":{
+                    "bc1qc3qcxs025ka9l6qn0q5cyvmnpwrqw2z49qwrx5":{
+                      "amount":"0.5",
+                      "usd_value":"3770.075"
+                    },
+                    "bc1qr4r8vryfzexvhjrx5fh5uj0s2ead8awpqspqra":{
+                      "amount":"0.5",
+                      "usd_value":"3770.075"
+                    }
+                  }
+                }
+              ]
+            },
+            "ETH":{
+              "0x78b0AD50E768D2376C6BA7de33F426ecE4e03e0B":{
+                "assets":{
+                  "ETH":{
+                    "amount":"10",
+                    "usd_value":"1650.53"
                   },
-                  "liabilities": {
-                      "eip155:1/erc20:0x6B175474E89094C44Da98b954EedeAC495271d0F": {"amount": "20", "usd_value": "20.35"}
+                  "eip155:1/erc20:0x6B175474E89094C44Da98b954EedeAC495271d0F":{
+                    "amount":"15",
+                    "usd_value":"15.21",
+                    "breakdown":{
```

Comment:
Question. Why a breakdown is needed if user's owned asset belongs only to one chain?